src/mrmimeparser.c

```diff
@@ -3,6 +3,7 @@
  *                              Delta Chat Core
  *                      Copyright (C) 2017 Björn Petersen
  *                   Contact: r10s@b44t.com, http://b44t.com
+ *                   Contact: schneider17@gmx.de (cs)
```

Comment:
as mentioned, the header must be equal over all files.

Reply:
You may change that?